projects/vdk-core/tests/functional/ingestion/jobs/test-ingest-bad-payload-job/1_ingest_step.py

```diff
@@ -15,8 +15,8 @@ def run(job_input: IJobInput):
     payload = payload_str
     if payload_str == "None":
         payload = None
-    elif payload_str == "date":
```

Comment:
I don't understand why you changed this to logger ?

Reply:
You have to pass an object that's not json-parseable. It was the easiest thing to pass.